Minor documentation updates (#464)
pa11y/pa11y: README.md

});
```

> If you need a GUI you can try [Koa11y](https://open-indy.github.io/Koa11y/), which is a desktop application that uses pa11y to run accessibility tests.

Reviewer: ```suggestion
If you need a GUI, you can try [Koa11y](https://open-indy.github.io/Koa11y/). It's a desktop application for Windows, OSX and Linux that uses Pa11y to run accessibility tests.
```
